### PR TITLE
SLSA version switch BuildFlag

### PR DESCRIPTION
**Pull request by adrianschroeter**

### CODE REVIEW

File: xml/obs_build_config.xml [hunk 1]
```diff
@@ -306,6 +306,16 @@ Support: pax debbuild</screen>
         </para>
        </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><parameter>slsaversion:VERSION</parameter></term>
+       <listitem>
+        <para>
+	  OBS 2.11 is producing slsa provenance files in version 0 by default at build
+	  time, when enabled.
```
Comment (hennevogel) [inline xml/obs_build_config.xml:314]: When enabled how? I guess by enabling `sbom:...`  right?

**Comment by adrianschroeter** [comment]
On Mittwoch, 10. Juli 2024, 11:48:33 CEST Henne Vogelsang wrote:
> @hennevogel commented on this pull request.
> 
> 
> 
> > @@ -306,6 +306,16 @@ Support: pax debbuild</screen>
>          </para>
>         </listitem>
>        </varlistentry>
> +      <varlistentry>
> +        <term><parameter>slsaversion:VERSION</parameter></term>
> +       <listitem>
> +        <para>
> +	  OBS 2.11 is producing slsa provenance files in version 0 by default at build
> +	  time, when enabled.
> 
> When enabled how? I guess by enabling `sbom:...`  right?

no, BSConfig, so this is only an admin option.

-- 

Adrian Schroeter ***@***.***>
Build Infrastructure Project Manager

SUSE Software Solutions Germany GmbH, Frankenstraße 146, 90461 Nürnberg, Germany
(HRB 36809, AG Nürnberg) Geschäftsführer: Ivo Totev

**Comment by hennevogel** [comment]
> no, BSConfig, so this is only an admin option.

You should tell that to people then...

**Comment by adrianschroeter** [comment]
well, yes, but not here. that needs to be an own chapter in admin guide.

**Pull request merged**